hawc/apps/epiv2/models.py

```diff
@@ -0,0 +1,143 @@
+    participant_n = models.PositiveIntegerField()
+    criteria = models.ManyToManyField(Criteria, blank=True)
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class Exposure(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="exposures"
+    )
+    measurement_type = models.ManyToManyField(MeasurementType)
+    measurement_timing = models.CharField()
+    exposure_route = models.CharField()
+    analytic_method = models.CharField()
+    comments = models.CharField()
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class ExposureLevel(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="exposure_levels"
+    )
+    chemical = models.ForeignKey(Chemical, on_delete=models.CASCADE)
+    exposure_measurement = models.ForeignKey(Exposure, on_delete=models.CASCADE)
+    sub_population = models.CharField()
+    # TODO: should some of this be separated into another model?
+    central_tendency = models.CharField()
+    central_tendency_type = models.CharField()
+    minimum = models.FloatField()
+    maximum = models.FloatField()
+    percentile25 = models.FloatField()
+    percentile75 = models.FloatField()
+    neg_exposure = models.FloatField()
+    comments = models.CharField()
+    data_location = models.CharField()
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class Outcome(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="outcomes"
+    )
+    health_outcome = models.CharField()
+    health_outcome_system = models.CharField()
+    measurement_timing = (
+        models.CharField()
+    )  # TODO: prototype had outcome measurement timing as a separate table, is that necessary?
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class AdjustmentFactor(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="adjustment_factors"
+    )
+    description = models.CharField()
+
+
+class DataExtraction(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="data_extractions"
+    )
+    outcome = models.ForeignKey(Outcome, on_delete=models.SET_NULL)
+    exposure = models.ForeignKey(Exposure, on_delete=models.SET_NULL)
+    n = models.PositiveIntegerField()
+    effect_estimate = models.CharField()
+    effect_estimate_type = models.CharField()
+    effect_description = models.CharField()
+    exposure_rank = models.PositiveSmallIntegerField()
+    # TODO: what are these fields
+    # CI LCL
```

Comment:
confidence interval; lower confidence level; upper confidence level, standard deviation, standard error. they should all be optional float fields